pkg/antctl/raw/multicluster/delete/clusterclaim.go

```diff
@@ -52,7 +52,7 @@ func (o *clusterClaimOptions) validateAndComplete() error {
 func NewClusterClaimCmd() *cobra.Command {
 	command := &cobra.Command{
 		Use:     "clusterclaims",
-		Args:    cobra.MaximumNArgs(1),
+		Args:    cobra.MaximumNArgs(0),
```

Comment:
Why did you remove the argument? I suppose allow user to remove `id.k8s.io` and `clusterset.k8s.io` separately.  @hjiajing

Reply:
Synced with Jiajing on Monday, I think this arg is not used? and Jiajing also thought this argument is not useful?